libs/blocks/quiz-entry/quiz-entry.css

```diff
@@ -1,142 +1,238 @@
+  background-position: center right 24px;
+  border: 2px solid #e6e6e6;
+  border-radius: 40px;
+  color: var(--text-color);
+  cursor: pointer;
+  font-size: 16px;
+  height: 56px;
+  line-height: var(--type-heading-m-lh);
+  padding: 0 var(--spacing-xxl) 0 var(--spacing-s);
+  width: 100%;
+}
+
+.quiz-input::placeholder {
+  color: var(--text-color);
   font-style: italic;
-  color: #2C2C2C;
 }
 
-input:disabled {
+.quiz-input:focus-visible {
+  outline: 4px solid #4B75FF;
+  outline-offset: 2px;
+}
+
+.quiz-input:disabled {
   background-color: #fafafacc;
   cursor: not-allowed;
   outline: none;
   opacity: 0.5;
 }
 
-.ml-clear-icon {
-  background-color: transparent;
-  transform: scale3d(1.4,1.4,1.4);
-  background-image: url("data:image/svg+xml,%0A%3Csvg id='Layer_1' data-name='Layer 1' xmlns='http://www.w3.org/2000/svg' viewBox='0 0 20 20'%3E%3Crect id='Frame' x='4.14' y='3.77' width='12' height='12' style='fill:red;fill-opacity:0'/%3E%3Cg id='Line'%3E%3Cpath d='M14.88,13.16a1,1,0,0,1,0,1.35,1,1,0,0,1-.67.27.94.94,0,0,1-.67-.27l-3.4-3.4-3.39,3.4a1,1,0,0,1-1.35,0,1,1,0,0,1,0-1.35L8.8,9.77,5.4,6.37A1,1,0,0,1,5.4,5,1,1,0,0,1,6.75,5l3.39,3.39L13.54,5a.95.95,0,1,1,1.34,1.34l-3.39,3.4Z' style='fill:%236e6e6e'/%3E%3C/g%3E%3C/svg%3E");
-  display: block;
-  position: absolute;
-  right: 13px;
-  top: calc(50% - 16px);
-  cursor: pointer;
-  width: 32px;
-  height: 32px;
+.quiz-input-clear {
+  background: url("data:image/svg+xml,%0A%3Csvg id='Layer_1' data-name='Layer 1' xmlns='http://www.w3.org/2000/svg' viewBox='0 0 20 20'%3E%3Crect id='Frame' x='4.14' y='3.77' width='12' height='12' style='fill:red;fill-opacity:0'/%3E%3Cg id='Line'%3E%3Cpath d='M14.88,13.16a1,1,0,0,1,0,1.35,1,1,0,0,1-.67.27.94.94,0,0,1-.67-.27l-3.4-3.4-3.39,3.4a1,1,0,0,1-1.35,0,1,1,0,0,1,0-1.35L8.8,9.77,5.4,6.37A1,1,0,0,1,5.4,5,1,1,0,0,1,6.75,5l3.39,3.39L13.54,5a.95.95,0,1,1,1.34,1.34l-3.39,3.4Z' style='fill:%236e6e6e'/%3E%3C/g%3E%3C/svg%3E") center no-repeat #fff;
```

Comment:
please add same position like search icon, same distance to the right side

Reply:
The position has been increased so it matches the search icon position more closely.